aws-rotate-keys/orb.yml

```diff
@@ -43,4 +43,20 @@ commands:
             curl --fail https://circleci.com/api/v1.1/project/github/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/envvar?circle-token=<< parameters.circleci-token >> \
               -X POST --header "Content-Type: application/json" \
               -d "{ \"name\":\"<< parameters.aws-secret-access-key-var >>\", \"value\":\"${new_secret_access_key}\" }" &&
-            aws iam delete-access-key --user-name << parameters.aws-username >> --access-key-id ${old_access_key_id}
+            aws iam delete-access-key --user-name << parameters.aws-username >> --access-key-id ${old_access_key_id} &&
+            echo &&
+            # Although the new token is visible on the aws API, aws does not accept it
+            # for authentication immediately.
+            echo 'export AWS_ACCESS_KEY_ID=$new_access_key_id' >> $BASH_ENV &&
+            echo 'export AWS_SECRET_ACCESS_KEY=$new_secret_access_key' >> $BASH_ENV &&
+            source $BASH_ENV &&
```

Comment:
Can you export the above directly without echoing them and then sourcing BASH_ENV?

Reply:
Good catch, I've fixed it.